CMakeLists.txt

```diff
@@ -154,9 +164,17 @@ if (BUILD_SHARED_LIBS)
                                              SOVERSION ${OPENTRACING_VERSION_MAJOR})
   target_compile_definitions(opentracing PRIVATE OPENTRACING_EXPORTS)
   install(TARGETS opentracing EXPORT OpenTracingTargets
-          RUNTIME DESTINATION ${LIB_INSTALL_DIR}
-          LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-          ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
+          RUNTIME DESTINATION ${LIB_INSTALL_DIR}64
+          LIBRARY DESTINATION ${LIB_INSTALL_DIR}64
+          ARCHIVE DESTINATION ${LIB_INSTALL_DIR}64
+          )
+  install(FILES ${CMAKE_BINARY_DIR}/output/libopentracing_mocktracer.so.${OPENTRACING_VERSION_STRING}
```

Comment:
Using `.so` like that is going to be a problem on OSX and Windows.

Could you add a `make install` command [here](https://github.com/opentracing/opentracing-cpp/blob/master/.travis.yml#L50) so that we have test coverage of the installation? and then fix the OSX error.

Reply:
I made the two changes suggested in the comments above but could you clarify what you mean here.

Reply:
OSX doesn't use `.so` for shared libraries, when you run `make install` it gives this error
```
-- Installing: /usr/local/lib64/libopentracing.dylib
CMake Error at cmake_install.cmake:55 (file):
  file INSTALL cannot find
  "/tmp/opentracing-cpp/build/output/libopentracing_mocktracer.so.1.5.0".


make: *** [install] Error 1
```